elastic_lib_client.py

```diff
@@ -0,0 +1,100 @@
+from elasticsearch import Elasticsearch
+import json
+import re
+import traceback
+from bs4 import BeautifulSoup
+from pathlib import Path
+from typing import Any
+
+
+class ElasticLibClient:
+    def __init__(
+        self,
+        base_url: str,
+        index_name: str = "index",
+        api_val: str = "",
+        user_id: str = "",
+    ) -> None:
+        self.INDEX = index_name
+        self.ELASTIC_SEARCH = Elasticsearch(base_url, api_key=(user_id, api_val))
+
+    def send_libguide(self, libguide: dict):
+        libguide["text"] = " ".join(libguide["text"])
+
+        es_doc_json = self._create_es_document(libguide)
+
+        op = self.ELASTIC_SEARCH.index(index=self.INDEX, document=es_doc_json)
```

Comment:
`flake8` points out that `op` is never used.  Add exception handling / logging as needed.

Reply:
Removed variable assignment